eth_keyfile/keyfile.py

```diff
@@ -215,9 +214,9 @@ def _create_v3_keyfile_json(
     else:
         raise EthKeyfileNotImplementedError(f"KDF not implemented: {kdf}")
 
-    iv = big_endian_to_int(Random.get_random_bytes(16))
+    iv_bytes = Random.get_random_bytes(16)
```

Comment:
I believe we want this to be a 32 byte string?

Reply:
16 bytes converted to a hex string has length 32. The issue filed has that incorrect.

From: https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition

> The key for the cipher is the leftmost 16 bytes of the derived key, i.e. DK[0..15]